web/src/OverviewSidebarOptions.tsx

```diff
@@ -260,13 +205,10 @@ export function OverviewSidebarOptions(
 ): JSX.Element {
   return (
     <OverviewSidebarOptionsRoot>
-      <OverviewSidebarOptionsButtonsRoot
-        className={!props.showFilters ? "is-filterButtonsHidden" : ""}
-      >
-        {props.showFilters ? filterOptions(props) : null}
+      <OverviewSidebarOptionsButtonsRoot className="is-filterButtonsHidden">
         <AlertsOnTopToggle
           className={props.options.alertsOnTop ? "is-enabled" : ""}
-          onClick={(e) => setAlertsOnTop(props, !props.options.alertsOnTop)}
+          onClick={(_e) => setAlertsOnTop(props, !props.options.alertsOnTop)}
```

Comment:
nit: I think it's more common in our codebase to do `(_) =>` or just `() =>`, but I don't know if there's a wider JS convention

Reply:
that is good to know! i think the wider convention that i'm used to is putting an `_` before variable names that are not used (but available).